src/sentry/static/sentry/app/views/eventsV2/utils.tsx

```diff
@@ -318,6 +328,9 @@ export function downloadAsCsv(tableData, columnOrder, filename) {
   link.setAttribute('download', `${filename} ${getUtcDateString(now)}.csv`);
   link.click();
   link.remove();
+
+  // Make testing easier
```

Comment:
I couldn't figure out how to keep using locationSpy for this since this doesn't change the url anymore with the download attribute being used instead. (note the download test is mostly ripped from the old download function)